changelog/5275.feature.rst

```diff
@@ -0,0 +1 @@
+Allows :meth:`sunpy.map.GenericMap.draw_quadrangle` to accept ``u.pix`` as input to enable plotting boxes in the pixel space of both the native map and other maps when the correct transform is specified.
```

Comment:
\`\`u.pix\`\` should instead be an intersphinx reference (particularly because `u` has no meaning here)